static/src/javascripts/projects/commercial/modules/ipsos-mori.ts

```diff
@@ -1,20 +1,17 @@
+// DCR documentation https://git.io/Jy5w8
+
 import { getConsentFor } from '@guardian/consent-management-platform';
-import { getLocale, loadScript } from '@guardian/libs';
+import { getLocale, loadScript, log } from '@guardian/libs';
 import { getInitialConsentState } from 'commercial/initialConsentState';
 import config from '../../../lib/config';
-import { log } from '@guardian/libs';
+import { stub } from './vendor/ipsos-mori';
 
 const loadIpsosScript = () => {
-	window.dm = window.dm || { AjaxData: [] };
-	window.dm.AjaxEvent = (et, d, ssid, ad) => {
-		dm.AjaxData.push({ et, d, ssid, ad }); // eslint-disable-line no-undef
-		if (window.DotMetricsObj) {
-			DotMetricsObj.onAjaxDataUpdate(); // eslint-disable-line no-undef
-		}
-	};
+	stub();
+
 	const ipsosSource = `https://uk-script.dotmetrics.net/door.js?d=${
 		document.location.host
-	}&t=${config.get('page.ipsosTag')}`;
+	}&t=${config.get('page.ipsosTag') as string}`;
```

Comment:
You can drop the assertion if you access the config directly and add the type to `global.d.ts`

```suggestion
	}&t=${window.guardian.config.page.ipsosTag}`;
```

or alternatively

```suggestion
	}&t=${config.get<string>('page.ipsosTag')}`;
```

Reply:
Good call - I'll go with passing the type to `config.get` since it seems as tough `config` is meant to have properties accessed via `get` rather than directly. This might be worth revisiting but probably not in this PR.

Reply:
If you use the `get<string>` approach, you’ll have to provide a fallback, otherwise you’ll get an error in your template literal. This is because there’s always a risk that the value is undefined. Your default should probably be an empty string.